App center documentation (#238)
exoplatform/exo-documentation: docs/PersonalApplications.rst

			like on your activity stream page. However, only your connections
			can see the Activity Composer that enables them to post on your stream.

.. _Managing-Fav-Apps:

=======================================
Managing your favourite applications
=======================================

Starting with eXo Platform 6, you can quickly access your favourite applications 

> through the application center drawer.

Reviewer: ```suggestion
through the Application Center drawer.
```

To access your favourite applications, you just need to click on this button |AC_image0| from
the top navigation bar.

|AC_image1|

> A drawer will open displaying your favourite applications alphabetically-organized in ascendant order.

Reviewer: ```suggestion
A drawer will open displaying your favourite applications alphabetically sorted
```

|AC_image2|
 
.. note:: When an administrator sets some applications as default, they also appear in your applications
          center drawer.

> When you click on an application icon it will open in the same tab of it is an eXo application, otherwise,

Reviewer: ```suggestion
When you click on an application icon it will open in the same tab of it is an eXo Platform, otherwise,
```

			like on your activity stream page. However, only your connections
			can see the Activity Composer that enables them to post on your stream.

.. _Managing-Fav-Apps:

=======================================
Managing your favourite applications
=======================================

Starting with eXo Platform 6, you can quickly access your favourite applications 
through the Applications drawer.

To access your favourite applications, you just need to click on this button |AC_image0| from
the top navigation bar.

|AC_image1|

A drawer will open displaying your favourite applications alphabetically sorted.

|AC_image2|
 
You can reorganize your favorite applications by dragging and dropping their icons according to your choice.

|AC_image3|
 

> .. note:: When an administrator sets some applications as default, they also appear in your applications

Reviewer 2: ```suggestion
.. note:: When an administrator sets some applications as mandatory, they also appear in your applications
```
The "default" option has disappear : https://docs.google.com/document/d/1fRKeiW7kYLMEOdFrL3AWPyJ3k_egp4AwCzdHt8FqRaM/edit#bookmark=id.yj1a1chsqm6y
and is replace by "mandatory"

Author: In fact, I planned to put it as "mandatory" when we implement it

          center drawer.

> When you click on an application icon it will open in the same tab of it is an eXo Platform, otherwise,

Reviewer 2: ```suggestion
When you click on an application icon it will open in the same tab of it is an eXo Platform application, otherwise,
```

Author: it was Patrice's suggestion here https://github.com/exoplatform/exo-documentation/pull/238#discussion_r420241896

Reviewer 2: So, I'm sure to understand what is "an eXo Platform"

Reviewer 2: Here I think we should put :
When you click on an application icon it will open in the same tab of it is an eXo Platform application, otherwise,

it will open in a new tab.

In addition to your favorite application displayed in the drawer, you can also navigate through the whole 
applications list you are allowed to view.

You just need to click on |AC_image4| button in the drawer to access the applications center.

|AC_image5|

> Application center is devided into 2 parts: on at the right listing your favorite applications and 

Reviewer 2: "Application Center" or "Applications Center"
Choose one :)

Author: Yep, I should align the nomination IMO Application center would be better, I'll align all in another commit

> You just need to click on |AC_image4| button in the drawer to access the applications center.
> 
> |AC_image5|
> 
> Application center is devided into 2 parts: on at the right listing your favorite applications and 

Reviewer 2: ```suggestion
You just need to click on |AC_image4| button in the drawer to access the applications center.

|AC_image5|

Application center is divided into 2 parts: on the right listing your favorite applications and 
```

> Application center is devided into 2 parts: on at the right listing your favorite applications and 

Reviewer 2: ```suggestion
Application center is devided into 2 parts: on at the right listing your favorite applications and 
```
favorite or favorites ? your or yours ?

Author: I think it does not take plural form the "favorite" adjective

> the second listing the whole applications your are allowed to view and use.

Reviewer 2: ```suggestion
on the the left listing the whole applications your are allowed to view and use.
```

This second part contains applications cards with these details for each one:

-  Application's name and image
-  Application's description
-  Two buttons "Open" and |AC_image6| in the card's bottom
-  The icon |AC_image7| in the card's topper

You can favorite an application in the application center by clicking on the |AC_image8| in the application's card.
The "Star" button will then be yellow colored and the application will be automatically added to the right panel
i.e. to the "favorite applications" list and to the drawer.

> You can also unfavorite an application in the applications center either by clicking on the yellow colored 

Reviewer 2: ```suggestion
You can also unfavorite an application in the applications center either by clicking on the yellow colored 
```
"applications center" or "Application Center" ?

Author: Application Center, I'll add another commit to align nominations

"Star" button on the applications card or on the "Star" button |AC_image9| in front of the application's name in the
"favorite applications" list.

> .. note:: When you reach the maximum of allowed favorite defined by an administrator, a red colored message
>           appears informing you that you are not allowed to add another favorite application.

Reviewer 2: ```suggestion
.. note:: When you reach the maximum of allowed favorite defined by an administrator, a red colored message appears informing you that you are not allowed to add another favorite application.
```

        However, if you have :ref:`made public <MakingPublic>` for your own wiki, anyone can access, read and edit it via the link.

.. _Managing-Fav-Apps:

=======================================
Managing your favorite applications
=======================================

> Starting with eXo Platform 6, you can quickly access your favourite applications 

Reviewer 2: favourite or favorite ? 
sometimes you use favourite, sometimes you use favorite. It should be consistent

Author: Ok I'll change to favorite

through the Applications drawer.

To access your favourite applications, you just need to click on this button |AC_image0| from
the top navigation bar.

|AC_image1|

A drawer will open displaying:

-  Mandatory applications defined by an administrator pinned in the drawer, alphabetically sorted but not draggable

-  Your favorite applications alphabetically sorted, by default, could be dragged and dropped to define your own choice.

|AC_image2|

.. note:: A separation line is present to differentiate mandatory applications from your favorite ones.
 
As cited above, you can reorganize your favorite applications by dragging and dropping their icons according to your choice.

> |AC_image3|

Reviewer 2: The visual of this animation does not show the drag and drop effect. Can we change to something clearer ?

Author: I used an application to make it, I don't know if we can make it better

> When you click on an application icon it will open in the same tab if it is an eXo Platform, otherwise,

Reviewer 2: ```suggestion
When you click on an application icon it will open in the same tab if it is an eXo Platform application, otherwise,
```

|AC_image1|

A drawer will open displaying:

> -  Mandatory applications defined by an administrator pinned in the drawer, alphabetically sorted but not draggable

Reviewer 2: ```suggestion
-  Mandatories applications defined by an administrator pinned in the drawer, alphabetically sorted and not draggable
```

> -  Your favorite applications alphabetically sorted, by default, could be dragged and dropped to define your own choice.

Reviewer 2: ```suggestion
-  Yours favorites applications alphabetically sorted, by default, could be dragged and dropped to define your own choice.
```

|AC_image2|

> .. note:: A separation line is present to differentiate mandatory applications from your favorite ones.

Reviewer 2: ```suggestion
.. note:: A separation line is present to differentiate mandatories applications from your favorites ones.
```

> As cited above, you can reorganize your favorite applications by dragging and dropping their icons according to your choice.

Reviewer 2: ```suggestion
As cited above, you can reorganize yours favorites applications by dragging and dropping their icons according to your choice.
```

|AC_image5|

> Application center is divided into 2 parts: on the right listing your favorite applications with mandatory applications pinned 

Reviewer 2: ```suggestion
Application center is divided into 2 parts: on the right listing yours favorites applications with mandatories applications pinned 
```

> in the top and on the the left listing the whole applications your are allowed to view and use.

Reviewer 2: ```suggestion
in the top and on the the left listing the whole applications you are allowed to view and use.
```

> .. note:: Mandatory applications coudn't be unfavorized and their star button is grey colored and unclickable in both parts: 

Reviewer 2: ```suggestion
.. note:: Mandatories applications coudn't be unfavorized and their star button is grey colored and unclickable in both parts: 
```

>           Favorite applications list and Application Center. 

Reviewer 2: ```suggestion
          Favorites applications list and Application Center. 
```

          On the grey star button's hover, a tooltip appears with the information "Mandatory application".
          
          |AC_image11|

The second part contains applications cards with these details for each one:

-  Application's name and image
-  Application's description
-  Two buttons "Open" and |AC_image6| in the card's bottom
-  The icon |AC_image7| in the card's topper

.. _star-button:

.. tip:: The star button contains three statuses:

>          - Grey color, not clickable: for mandatory applications

Reviewer 2: ```suggestion
         - Grey color, not clickable: for mandatories applications
```

>          - Yellow borders and white body: for application not yet favorized

Reviewer 2: ```suggestion
         - Yellow borders and white body: for applications not yet favorized
```
